src/model/Computation.js

```diff
@@ -157,4 +157,13 @@ export default class Computation extends Model {
 
 		this.dependencies = dependencies;
 	}
+
+	teardown () {
+		let i = this.dependencies.length;
+		while ( i-- ) {
+			if ( this.dependencies[i] ) this.dependencies[i].unregister( this );
+		}
+		if ( this.root.computations[this.key] === this ) delete this.root.computations[this.key];
```

Comment:
From [v8 forum](https://groups.google.com/forum/#!topic/v8-users/zE4cOHBkAnY):

> We only track property additions, because these are by far the most common case. Deleting a property results in going to "slow mode", i.e. using a dictionary for the object's properties. So as a general rule of thumb, using 'delete' makes thing slower.

> When you are using an object as a hash map, using delete is OK, because then the object is very probably already in dictionary (= hash map) mode. Playing GC by hand is not necessary and almost always the wrong thing, anyway.